src/impact-dashboard/MitigationInformation.tsx

```diff
@@ -127,7 +127,6 @@ const MitigationInformation: React.FC = () => {
         <FormGridRow>
           <FormGridCell>
             <InputSelect
-              label="Rate of spread"
```

Comment:
Seeking Feedback on this. It looks like, in addition to the facility page,  this `MitigationInformation` is only being used on the '/inspect-model' page.  Are we eventually going to move this whole section towards the styling/layout in the current mock? If yes, then I think we can continue to make changes directly in this component here. Otherwise, if we want the two pages to maintain two separate layouts, I will work on compartmentalizing the change and creating a separate mitigation information component for the facility page.

Reply:
I think @macfarlandian is the right person to chime in on this question.

Reply:
that's fine, we don't care much about the styling and layout of inspect-model as long as it remains functional